{"name": "test_attendance_outside_sign_back[circle0]", "status": "broken", "statusDetails": {"message": "KeyError: 'invoker'", "trace": "self = <intention_test.TestKaoQin object at 0x00000287ACCA28B0>\ncircle = {'text': '外勤签退'}\n\n    @allure.story('考勤管理>打卡>外勤签退')\n    @pytest.mark.parametrize('circle', attendance_outside_sign_back)\n    def test_attendance_outside_sign_back(self, circle):\n        # 请求接口并获取返回的data\n        request = request_get_ceshi(circle['text'])\n        response = json.loads(request['data'])\n        # 接口返回的值\n        response_title = response['intentInfo']['title']\n        response_name = response['intentInfo']['name']\n        response_text = response['text']\n        response_userId = response['userId']\n        response_status = response['status']\n>       response_action = response['invoker']['action']\nE       KeyError: 'invoker'\n\nTestCase\\intention_test.py:116: KeyError"}, "parameters": [{"name": "circle", "value": "{'text': '外勤签退'}"}], "start": 1679538644324, "stop": 1679538645554, "uuid": "9362af65-212e-4fdd-9a30-952581538caf", "historyId": "0c85af30c8dccb609c007588784f89eb", "testCaseId": "fc196a50e497d72b173688dc467b298e", "fullName": "TestCase.intention_test.TestKaoQin#test_attendance_outside_sign_back", "labels": [{"name": "feature", "value": "考勤管理"}, {"name": "story", "value": "考勤管理>打卡>外勤签退"}, {"name": "epic", "value": "项目名称：泛微小e意图识别"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "intention_test"}, {"name": "subSuite", "value": "TestKaoQin"}, {"name": "host", "value": "DESKTOP-BAUEVGC"}, {"name": "thread", "value": "20684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.intention_test"}]}